docs/herbarium/getting-started.md

```diff
@@ -0,0 +1,40 @@
+---
+id: gettingStarted
+title: Getting Started with Herbarium
+sidebar_label: Getting Started
+slug: /herbarium/getting-started
+---
+
+### Herbarium
+
+Herbarium is a centralized and standardized repository and discovery service for Herbs objects (entities, use cases, repositories) allowing glues (ex: Herbs Shelf, herbs2rest, herbs2gql, herbs2knex, etc) to access, explore and find these objects.
+
+### Installing
+
+```javascript
+$ npm install @herbsjs/herbarium
+```
+
+### Initializing
```

Comment:
Suggested text:
To start using Herbarium it is necessary to load all the objects in the project.

Reply:
what if:
"To start using Herbarium it is necessary that all project objects are built and then load them."

Reply:
> all project objects are built 

what do you mean by that?